extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -227,12 +227,12 @@ public long getTime(String hostname) {
         } catch (UnknownHostException uhe) {
             String msg = getThing().getUID().toString() + " the given hostname '" + hostname
                     + "' of the timeserver is unknown -> returning current sytem time instead.";
-            logger.warn(msg);
-            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.COMMUNICATION_ERROR, msg);
+            logger.debug(msg);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, msg);

```

Comment:
Did you change this by intention? Then why did you leave it untouched for an IOException? The result effectively is the same, so I think it should be OFFLINE then too.

As the JavaDoc above of this method currently states, this handler is implemented in a way that it fall back to the current system time if no connection could be established. IMHO this is awkward, so I'd second your change to switch it to OFFLINE/COMMUNICATION_ERROR - but do it consistently then.